runtime/parser/function.go

```diff
@@ -185,10 +185,13 @@ func parseFunctionDeclaration(
 	accessPos *ast.Position,
 	purity ast.FunctionPurity,
 	purityPos *ast.Position,
+	staticPos *ast.Position,
+	nativePos *ast.Position,
```

Comment:
Now that we have 4 of these, it might be nice to refactor this at some point into a `DeclarationPrefixes` struct or something similar just to cut down on the sheer number of arguments in places like this.